### PR TITLE
accommodate proofs with repeat cred defs, check for bait-and-switch b…

### PR DESCRIPTION
**Pull request by sklump**
…etween presentation proposal and request

Signed-off-by: sklump <srklump@hotmail.com>

### CODE REVIEW

File: aries_cloudagent/protocols/present_proof/v1_0/routes.py [hunk 1]
```diff
@@ -427,61 +427,6 @@ class V10PresentationRequestSchema(Schema):
     return web.json_response(presentation_exchange_record.serialize())
```
Comment (andrewwhitehead) [inline aries_cloudagent/protocols/present_proof/v1_0/routes.py]: I'm not sure if `presentation_exchange_create_request` was removed intentionally, but I believe we use that in identity kit to create presentation exchanges that can be responded to without a connection.

**Comment by swcurran** [comment]
We definitely need to be able to support that use case.  @sklump - if something needs to be added to or clarified in the aries-RFC 0037, please let me know.

**Comment by sklump** [comment]
Sorry, I figured it was redundant with send...(). I will put it back and amend the PR Monday.

File: aries_cloudagent/protocols/present_proof/v1_0/routes.py (hunk 1 above)
Comment (sklump) [reply aries_cloudagent/protocols/present_proof/v1_0/routes.py]: My mistake, I had thought it redundant vs. ...send...(). I am fixing and amending the PR.

**Pull request merged**